src/AppInstallerCLICore/Commands/FeaturesCommand.cpp

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+#include "FeaturesCommand.h"
+#include "TableOutput.h"
+#include <winget/UserSettings.h>
+
+namespace AppInstaller::CLI
+{
+    using namespace Utility::literals;
+    using namespace AppInstaller::Settings;
+
+    Resource::LocString FeaturesCommand::ShortDescription() const
+    {
+        return { Resource::String::FeaturesCommandShortDescription };
+    }
+
+    Resource::LocString FeaturesCommand::LongDescription() const
+    {
+        return { Resource::String::FeaturesCommandLongDescription };
+    }
+
+    std::string FeaturesCommand::HelpLink() const
+    {
+        return "https://aka.ms/winget-seattings";
```

Comment:
>https://aka.ms/winget-seattings [](start = 16, length = 31)

Incorrect copy pasta

Reply:
This was intentional. Are we saying we will have a special md file/microsoft docs page for this hidden command? I was thinking settings.md will have in the experimentalFeatures entry something like "to know which experimental features are available run winget features"

---
In reply to: [446445866](https://github.com/microsoft/winget-cli/pull/460#discussion_r446445866) [](ancestors = 446445866)

Reply:
I guess the link can point to <github url>#experimentalFeatures to be clear

---
In reply to: [447183011](https://github.com/microsoft/winget-cli/pull/460#discussion_r447183011) [](ancestors = 447183011,446445866)